clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -191,6 +193,15 @@ class StandardGceAttributeValidator {
     return result
   }
 
+  def validateBetweenZeroAndOneDouble(double value, String attribute) {
```

Comment:
Why not `validateInRangeExclusive(double value, double min, double max, String attribute)`

Or just `validateBetweenZeroAndOne(double value, String attribute)`. 

Either way, don't include `Double` in the method name because Groovy supports [method overloading](http://beginnersbook.com/2013/05/method-overloading/).